frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -288,24 +291,27 @@ const createInferenceServiceAndDataConnection = (
         isModelMesh,
         acceleratorProfileState,
         replicaCount,
+        dryRun,
       )
     : createInferenceService(
         inferenceServiceData,
         undefined,
         isModelMesh,
         acceleratorProfileState,
         replicaCount,
+        dryRun,
       );
 };
 
-export const submitInferenceServiceResource = (
+// TODO mturley we need a dryRun here too right?
```

Comment:
I guess you forgot to remove this TODO right? I think returning the function with the dry run is a great approach here. Are you considering something else?

Reply:
Yeah I just forgot to remove this